pkg/util/converter/prom.go

```diff
@@ -981,7 +981,7 @@ func readCategorizedStream(iter *jsonitere.Iterator) backend.DataResponse {
 			case "stream":
 				// we need to clear `labels`, because `iter.ReadVal`
 				// only appends to it
-				labels := data.Labels{}
+				labels = data.Labels{}
```

Comment:
Wat. Using := is like definining a new `labels` here? I.e. `const labels = data.Labels` 😖

Reply:
Yep, it's declaration and assignment.